dev/TeachingTip/TeachingTip.xaml

```diff
@@ -494,6 +494,7 @@
                                       AutomationProperties.LandmarkType="Custom"
                                       Background="{TemplateBinding Background}"
                                       BorderBrush="{TemplateBinding BorderBrush}"
+                                      FlowDirection="{TemplateBinding FlowDirection}"
```

Comment:
>FlowDirection [](start = 38, length = 13)

I don't understand why FlowDirection is here instead of on TailOcclusionGrid and/or TopHighlight[Left|Right]

Reply:
Ultimately I think it should be. But there's some other logic updates that seem to be required that I want to wait for Stephen to return to talk through.